### PR TITLE
V5X Rev 2 updates

### PR DESCRIPTION
**Pull request by davids5**

### CODE REVIEW

File: boards/px4/fmu-v5x/init/rc.board_sensors [hunk 1]
```diff
@@ -55,17 +55,20 @@ else
 
 fi
 
+# External compass on GPS1/I2C1 (the 3rd external bus): standard Holybro Pixhawk 4 or CUAV V5 GPS/compass puck (with lights, safety button, and buzzer)
+ist8310 -X -b 1 -R 10 start
```
Comment (dagar) [inline boards/px4/fmu-v5x/init/rc.board_sensors:59]: Is I2C1 the special 10 pin connector on all v5x variants?

Comment (davids5) [reply boards/px4/fmu-v5x/init/rc.board_sensors:59]: > Is I2C1 the special 10 pin connector on all v5x variants?

I believe so

**Comment by davids5** [comment]
@dagar 1bbabb6190375ac07647953ee19ebda0c285fbd4 fixed the breakage I was seeing

**Comment by davids5** [comment]
@dagar - this needs to come in and be backported to the next stable point release. We will also need https://github.com/PX4/PX4-Autopilot/pull/17994 once I can verify the next ver of the INA228 HW. 

FYI @mrpollo

**Comment by mrpollo** [comment]
Thanks @davids5

**Review by dagar — APPROVED** [review]

**Pull request merged**